universal-application-tool-0.0.1/app/auth/UatProfile.java

```diff
@@ -71,4 +71,23 @@ public boolean isUatAdmin() {
   public String getId() {
     return profileData.getId();
   }
+
+  public CompletableFuture<Void> setEmailAddress(String emailAddress) {
+    return this.getAccount()
+        .thenApplyAsync(
+            a -> {
+              a.setEmailAddress(emailAddress);
+              a.save();
+              return null;
+            },
+            dbContext);
+  }
+
+  public CompletableFuture<String> getEmailAddress() {
+    return this.getAccount().thenApplyAsync(Account::getEmailAddress);
```

Comment:
Suggestion: I think this needs `HttpExecutionContext` to ensure the class loader stays available

Reply:
Done.